### ISSUE TITLE
Replace README.md badges with automated ones

### ISSUE THREAD

**Issue by ascillitoe** [SeldonIO/alibi#590]
#589 adds classifiers to setup.py to automate the README.md badges. However, these will not take effect until we publish a new PyPI release. Prior to release we should remove the hardcoded PyPI badges and replace them with the automated ones. 

For example:
```
![Python version](https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9-blue.svg)
```
should be replaced with:
```
[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/alibi?logo=pypi&style=flat&color=blue)][#pypi-package]
```

**Comment by ascillitoe** [comment]
Moved to `0.6.6` milestone so the classifiers to be released in `0.6.5` will be picked up.

**Comment by ascillitoe** [comment]
Addressed by https://github.com/SeldonIO/alibi/commit/5efc04b4963a39082e73f3b3a091133162494d0e
